docs/sql-reference/14-sql-commands/00-ddl/103-aggregating-index/create-aggregating-index.md

```diff
@@ -16,9 +16,11 @@ Creates a new aggregating index in Databend.
 ## Syntax
 
 ```sql
-CREATE AGGREGATING INDEX <index_name> AS SELECT ...
+CREATE [SYNC] AGGREGATING INDEX <index_name> AS SELECT ...
 ```
 
+- The `SYNC` parameter, when used during the creation of an aggregating index, enables automatic refresh, ensuring the index stays synchronized with the latest data updates.
```

Comment:
If my table already have lot data, does the `SYNC` keywork will build the index when the aggregating index creating? @ariesdevil